src/itest/java/com/iexec/worker/tee/post/compute/IntegrationTests.java

```diff
@@ -0,0 +1,94 @@
+/*
+ * Copyright 2022 IEXEC BLOCKCHAIN TECH
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.iexec.worker.tee.post.compute;
+
+
+import lombok.extern.slf4j.Slf4j;
+import org.awaitility.Awaitility;
+import org.junit.ClassRule;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.testcontainers.containers.DockerComposeContainer;
+import org.testcontainers.containers.GenericContainer;
+import org.testcontainers.containers.output.Slf4jLogConsumer;
+import org.testcontainers.junit.jupiter.Container;
+import org.testcontainers.utility.DockerImageName;
+
+import java.io.File;
+import java.util.concurrent.TimeUnit;
+
+@Slf4j
+class IntegrationTests {
+
+    public static final String RESOURCES_DIR = "src/itest/resources";
+    public static final String MOCK_WORKER_DOCKER_COMPOSE_YML =
+            RESOURCES_DIR + "/docker-compose.yml";
+    public static final String CONTAINERS_PREFIX = "iexec-";
+    public static final String WORKER = "worker";
+    public static final String POST_COMPUTE = "post-compute";
+    public static final String INTERNAL_NETWORK = "iexec-post-compute-net";
+    public static final String TEE_WORKER_POST_COMPUTE_IMAGE =
+            "nexus.iex.ec/tee-worker-post-compute:dev";
+
+    @ClassRule
```

Comment:
Is this still functional with Junit 5? Shouldn't it be `@Extension`?

Reply:
It is not clearly documented on testcontainers.org, but `@ClassRule` annotation is for JUnit4 as mentioned by @zguesmi . (https://www.testcontainers.org/quickstart/junit_4_quickstart/)
With JUnit5, you have to use the `@Container` annotation for single containers as well as for docker compose.
```suggestion
    @Container
```

Reply:
https://www.testcontainers.org/quickstart/junit_5_quickstart/